src/packages/api/bill/bill-query-builder.ts

```
@@ -0,0 +1,17 @@
+/**
+ * The Bill Query Builder
+ */
+export class BillQueryBuilder extends BaseQueryBuilder<Bill> {
+	/**
+	 * Constructor
+	 * @param endpoint - The Endpoint
+	 * @param baseQuery - The Base Query
+	 */
+	constructor(endpoint: string, baseQuery: Query) {
+		super(endpoint, baseQuery);
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)